### PR TITLE
fix: forbid export of labels for externalIds with slashes

### PR DESCRIPTION
**Pull request by Jonas1312**
e2e: https://github.com/kili-technology/kili-python-sdk/actions/runs/4627297169

### CODE REVIEW

File: src/kili/entrypoints/queries/label/__init__.py [hunk 1]
```diff
@@ -642,7 +642,7 @@ def export_labels(
             project_id: Identifier of the project.
             filename: Relative or full path of the archive that will contain
                 the exported data.
-            fmt: Format of the exported labels.
+            fmt: Format of the exported labels. One of "raw", "kili", "yolo_v4", "yolo_v5", "yolo_v7", "coco", "pascal_voc".
```
Comment (PierreLeveau) [inline src/kili/entrypoints/queries/label/__init__.py:645]: it is already highlighted through the `LabelFormat` type: 
image

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

File: src/kili/entrypoints/queries/label/__init__.py (hunk 1 above)
Comment (Jonas1312) [reply src/kili/entrypoints/queries/label/__init__.py:645]: image

it doesn't show in the vscode showHover

Comment (PierreLeveau) [reply src/kili/entrypoints/queries/label/__init__.py:645]: image

Comment (PierreLeveau) [reply src/kili/entrypoints/queries/label/__init__.py:645]: it shows the options when you start writing. But I agree that you have to know the trick.

Comment (Jonas1312) [reply src/kili/entrypoints/queries/label/__init__.py]: I removed it

**Review by PierreLeveau — APPROVED** [review]

**Pull request merged**